nvflare/private/fed/utils/fed_utils.py

```diff
@@ -214,3 +215,26 @@ def fobs_initialize():
     flare_decomposers.register()
     common_decomposers.register()
     private_decomposers.register()
+
+
+def set_stats_pool_config_for_job(workspace: Workspace, job_id: str):
+    job_meta = get_job_meta_from_workspace(workspace, job_id)
+    stats_config = job_meta.get("stats_pool_config")
```

Comment:
instead of using hard-coded string here "stats_pool_config"

We could add this into ./nvflare/apis/job_def.py "JobMetaKey"

Reply:
Yes, we should.